src/libopensc/pkcs15-pteid.c

```diff
@@ -106,22 +107,24 @@ static int sc_pkcs15emu_pteid_init(sc_pkcs15_card_t * p15card)
 	/* TODO: Use Portuguese descriptions? */
 	
 	/* Add X.509 Certificates */
-	for (i = 0; i < 4; i++) {
-		static const char *pteid_cert_names[4] = {
-				"AUTHENTICATION CERTIFICATE",
-				"SIGNATURE CERTIFICATE",
-				"SIGNATURE SUB CA",
-				"AUTHENTICATION SUB CA"
+	for (i = 0; i < 5; i++) {
+		static const char *pteid_cert_names[5] = {
+			"AUTHENTICATION CERTIFICATE",
+			"SIGNATURE CERTIFICATE",
+			"SIGNATURE SUB CA",
+			"AUTHENTICATION SUB CA",
+			"ROOT CA"
 		};
 		/* X.509 Certificate Paths */
-		static const char *pteid_cert_paths[4] = {
+		static const char *pteid_cert_paths[5] = {
 			"3f005f00ef09", /* Authentication Certificate path */
-			"3f005f00ef08", /* Digital Signature Certificate path */
+			"3f005f00ef08", /* Digital Signature Certificate path */			
 			"3f005f00ef0f", /* Signature sub CA path */
-			"3f005f00ef10"	/* Authentication sub CA path */
+			"3f005f00ef10",	/* Authentication sub CA path */
+			"3f005f00ef11"  /* Root CA path */
```

Comment:
I had NOOO idea that was usual, not even legal syntax. I see the rational for source control and I really like the idea. Thanks!